tests/scripts/all.sh

```diff
@@ -1088,6 +1088,12 @@ component_build_arm_none_eabi_gcc () {
     make CC=arm-none-eabi-gcc AR=arm-none-eabi-ar LD=arm-none-eabi-ld CFLAGS='-Werror -Wall -Wextra' lib
 }
 
+component_build_arm_none_eabi_gcc_armel () {
```

Comment:
Do you mean renaming to `arm5vte`? I called it `armel` because it's [close enough](https://github.com/ARMmbed/mbedtls/pull/2169#issuecomment-518178508) to Debian's armel platform, but I realize it isn't clear by reading the code. Would adding a comment do?

Reply:
Yes, that's the rename I'm suggesting. If you find the name accurate, then I don't think a comment is needed. I just couldn't find evidence of `armel` being exactly equivalent to `arm5vte`. I thought maybe, because of tunnel vision effect, the test has name has been mistakenly tied too closely to the issue at hand, and not named for what it actually does, but I don't think it's worth the effort of changing and re-reviewing, so I hold my approval.

Reply:
I renamed the component and added a comment to explain why it's there.